cupy/_io/formatting.py

```diff
@@ -52,3 +52,12 @@ def array2string(a, *args, **kwargs):
 
     """
     return numpy.array2string(cupy.asnumpy(a), *args, **kwargs)
+
+
+def format_float_positional(a, *args, **kwargs):
+    """Returns the decimal string in positional notation
+
```

Comment:
* Keep the same signature and docstring as NumPy.
* Add notes about arguments.

```suggestion
def format_float_positional(x, *args, **kwargs):
    """Format a floating-point scalar as a decimal string in positional
    notation.
    
    See :func:`numpy.format_float_positional` for the list of arguments.

```

Reply:
@kmaehashi, I had updated my PR with suggested changes. can you please review them. Thank you

Reply:
Added suggested changes.